tests/countries/test_india.py

```
@@ -10,989 +10,27 @@
-
 import warnings
 from unittest import TestCase
 
+from holidays.constants import OPTIONAL
 from holidays.countries.india import India
 from tests.common import CommonCountryTests
 
 
 class TestIndia(CommonCountryTests, TestCase):
     @classmethod
     def setUpClass(cls):
```

Ruff (0.15.10)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)